contracts/cheddar_5x5_tic_tac_toe/src/internal.rs

```diff
@@ -74,7 +75,7 @@ impl Contract {
                 );
             }
         }
-        self.last_update_timestamp = ts;
+        self.last_update_timestamp = nano_to_sec(ts);
     }
```

Comment:
I think we don't need this. Player should remove themselves manually from the queue if they want to get back the deposit.

Reply:
Same as above